tests/deepsparse/loggers/metric_functions/template_functions/test_template_functions.py

```diff
@@ -12,11 +12,163 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+import time
+
+import numpy
+
 import pytest
 from deepsparse import Pipeline
+from deepsparse.loggers.build_logger import logger_from_config
 from tests.utils import mock_engine
 
 
+@pytest.mark.parametrize(
+    "group_name, pipeline_name, inputs, expected_logs",
+    [
+        (
+            "image_classification",
+            "image_classification",
+            {"images": [numpy.ones((3, 224, 224))] * 2},
+            "tests/deepsparse/loggers/metric_functions/template_functions/template_logs/image_classification_logs.txt",  # noqa E501
```

Comment:
let's have a helper function that automatically generates this path using helper functions. (additionally it's best practice to use pathlib/os libraries for nesting directories since not every OS uses `/` to separate directories)